cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2830,6 +2830,103 @@ action.summary_index.category = Authentication
+| rename Authentication.* as * \
+| eval time = _time \
+| `cs_generate_time_fields_for_filter(time)` \
+| where time_wday="Saturday" OR time_wday="Sunday" OR time_hour<6 OR time_hour>19 \
+| `cs_remove_time_fields_for_filter(time)` \
+| iplocation src \
+| table time, user, action, count, src, City, Region, Country \
+| `cs_human_readable_time_format(time)` \
+| `cs_authentication_vpn_login_attemp_outside_working_hour_filter`
+action.summary_index = 1
+action.summary_index._name = cyences
+action.summary_index.category = Authentication
+action.summary_index.alert_name = Authentication - VPN Login Attemp Outside Working Hour
+
+
+[Authentication - Usual Login Location Lookup Gen]
+disabled = 0
+enableSched = 1
+alert.track = 0
+cron_schedule = 0 1 * * *
+description = Generates the cs_authentication_usual_location.csv file with the Usual Login Location details for users. \
+\
+Data Collection - VPN data mapped with authentication data-model and has dest_category=vpn_auth.
+dispatch.earliest_time = -7d@h
+dispatch.latest_time = now
+display.general.timeRangePicker.show = 0
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+display.visualizations.show = 0
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | tstats `cs_summariesonly_authentication` count from datamodel=Authentication where Authentication.dest_category="vpn_auth" AND Authentication.action="success" by Authentication.app, Authentication.user, Authentication.src \
+| `drop_dm_object_name(Authentication)` \
+| iplocation src \
+| stats sum(count) as count by user, app, Country \
+| eventstats sum(count) as total by user, app \
+| eval pct = round(count*100/total, 2) \
+| table user app Country count, total, pct \
+| outputlookup cs_authentication_usual_location.csv
+
+
+[Authentication - Successful VPN Login From New Location]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A Successful VPN Login From New Location. \
+\
+Data Collection - VPN data mapped with authentication data-model and has dest_category=vpn_auth.
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | tstats `cs_summariesonly_authentication` count from datamodel=Authentication where Authentication.dest_category="vpn_auth" AND Authentication.action="success" by Authentication.app, Authentication.user, Authentication.src, _time \
+| `drop_dm_object_name(Authentication)` \
+| iplocation src \
+| lookup cs_authentication_usual_location.csv app, user, Country OUTPUT pct \
+| fillnull pct \
+| where pct < `cs_authentication_new_location_login_pct_limit` \
```

Comment:
Instead of all this. My suggestion will be to notify when user is login from any new location. Instead of using PCT and all.

Reply:
Yes. we can do that. @VatsalJagani What do you think?

Reply:
This scenarios is already covered in the this case. Also if we want to just show new login then we can manually update the macro value as per requirement as per needs